src/Maker/MakeDockerDatabase.php

```diff
@@ -0,0 +1,121 @@
+        ;
+    }
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command): void
+    {
+        parent::interact($input, $io, $command);
+
+        $io->newLine();
+
+        $this->databaseChoice = $io->choice(
+            'Which database service will you be creating?',
+            ['MySQL', 'MariaDB', 'Postgres']
+        );
+
+        $io->text([sprintf(
+            'For a list of supported versions, check out https://hub.docker.com/_/%s',
+            strtolower($this->databaseChoice)
+        )]);
+
+        $suggestedVersion = DatabaseServices::getSuggestedServiceVersion($this->databaseChoice);
+
+        $this->serviceVersion = $io->ask('What version would you like to use?', $suggestedVersion);
+
+        $this->serviceName = 'database';
+
+        if ($this->composeFileManipulator->serviceExists($this->serviceName)) {
+            $this->showServiceAlreadyDefinedComment($io);
+
+            $randomName = str_replace(' ', '-', Str::getRandomTerm());
+
+            $envTest[] = 'If you are using the Symfony Binary, it will expose the env_var associated with this service.';
+            $envTest[] = sprintf('If you name the service %s, the binary will look for %s_URL', $randomName, strtoupper($randomName));
+
+            $io->text($envTest);
+
+            $this->serviceName = $io->ask(sprintf(
+                'What name should we call the new %s service? e.g. %s',
+                $this->serviceName,
+                $randomName
+            ));
+        }
+
+        $io->section(sprintf('- %s -', $this->databaseChoice));
+
+        $defaults = [
+            sprintf('Port(s) %s are exposed to the host.', ...(DatabaseServices::getDefaultPorts($this->databaseChoice))),
+        ];
+
+        $io->text($defaults);
+    }
+
+    public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator): void
+    {
+        $io->newLine();
+
+        $service = DatabaseServices::getDatabaseSkeleton($this->databaseChoice, $this->serviceVersion);
+
+        $this->composeFileManipulator->addDockerService($this->serviceName, $service);
+
+        $generator->dumpFile($this->composeFilePath, Yaml::dump($this->composeFileManipulator->getComposeData(), 20));
+        $generator->writeChanges();
+
+        $this->writeSuccessMessage($io);
+
+        $io->text(sprintf('The new <fg=yellow>"%s"</> service is now ready.', $this->serviceName));
+        $io->newLine();
+
+        $closing[] = 'Next:';
+        $closing[] = 'Run <fg=yellow>docker-compose up -d</> to start your database container.';
+        $closing[] = 'If you are using the Symfony Binary, it will detect the new service automatically.';
+        $closing[] = '';
+        $closing[] = sprintf('<fg=yellow>docker-compose stop %s</> will stop the container without removing it.', $this->serviceName);
+        $closing[] = '<fg=yellow>docker-compose down</> will stop and remove all containers in the compose file.';
```

Comment:
We should probably make these 2 consistent with either mentioning the service name or not. For me, the key thing is to say that "stop" stops and "down" fully destroys them. My preference would be to show both of them without the serviceName.

Also:

> will stop and destroy all of your containers.

Reply:
`docker-compose stop` will stop all of the containers listed in the compose file.
`docker-compose stop mysql` will only stop the container named "mysql" in the compose file.
`docker-compose down` does not accept any args (service names) and stops + destroys all of the containers in the compose file.. 

For `down` I think leaving off the `in the compose file.` might lead to confusion. "Hey i ran down in the project over there, why is this container still running over here?"

Reply:
Ok, then I vote for

> <>docker-compose stop</> will stop all the containers in docker-compose.yaml
> <>docker-compose down</> will stop *and* destroy the containers

WDYT? I mention the `docker-compose.yaml` in the first (you're right that it's clearer with it). Then, I think it's clear enough that we don't need to repeat in the second, but we could.

Reply:
done